hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestListParts.java

```diff
@@ -130,7 +131,7 @@ public void testListPartsWithUnknownUploadID() throws Exception {
       REST.get(OzoneConsts.S3_BUCKET, OzoneConsts.KEY, 0,
           uploadID, 2, "0");
     } catch (OS3Exception ex) {
-      Assert.assertEquals(S3ErrorTable.NO_SUCH_UPLOAD.getErrorMessage(),
+      assertEquals(S3ErrorTable.NO_SUCH_UPLOAD.getErrorMessage(),
```

Comment:
Same comment as in TestBucketPut.

Reply:
Created [HDDS-9760](https://issues.apache.org/jira/browse/HDDS-9760) to address the logical error in three test cases where the catch block is not getting executed.